# HG changeset patch -- Bitbucket.org
# Project textext
# URL http://bitbucket.org/pv/textext/overview
# User Pauli Virtanen <pav@iki.fi>
# Date 1237138023 -7200
# Node ID 2a376a0465a43f7492b901d0ad96fb47eadf4e17
# Parent  6a2e35ad799a179c147833a6ba828812958b8491
Avoid Python2.6 deprecation warning by using hashlib instead of md5

--- a/textext.py
+++ b/textext.py
@@ -52,9 +52,14 @@ sys.path.append(r'c:/Program Files/Inksc
 sys.path.append(os.path.dirname(__file__))
 
 import inkex
-import os, sys, tempfile, traceback, glob, re, md5, copy
+import os, sys, tempfile, traceback, glob, re, copy
 from lxml import etree
 
+try:
+    import hashlib
+except ImportError:
+    import md5 as hashlib
+
 USE_GTK = False
 try:
     import pygtk
@@ -871,7 +876,7 @@ class Pdf2Svg(PdfConverterBase):
 
     def convert(self, *a, **kw):
         # compute hash for generating unique ids for sub-elements
-        self.hash = md5.new('%s%s' % (a, kw)).hexdigest()[:8]
+        self.hash = hashlib.md5('%s%s' % (a, kw)).hexdigest()[:8]
         return PdfConverterBase.convert(self, *a, **kw)
 
     def pdf_to_svg(self):
